security/operator/secopspolicy/deploy/crds/defenition.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: secopspolicies.secops.pathfinder.gov.bc.ca
+spec:
+  group: secops.pathfinder.gov.bc.ca
+  names:
+    kind: SecOpsPolicy
+    listKind: SecOpsPolicyList
+    plural: secopspolicies
+    singular: secopspolicy
+  scope: Namespaced
```

Comment:
@jleach We are likely looking at a cluster-scoped operator here. Otherwise all rules will exist within a single namespace, which I don't think is what we want. My original thought was that the pattern would be for teams to place all of their CR objects in their `-tools` namespace, though in reality if this is cluster-scoped, it doesn't really matter where they place the object.

Reply:
@stewartshea I'm not sure just yet where it should go. I think I'd prefer to keep prod rules with the environment so that if I'm debugging I can just stay in one name space; it also makes my environments more stand alone.